sdk/provisioning/Azure.Provisioning.CloudMachine/src/OFX/StorageServices.cs

```diff
@@ -42,14 +42,50 @@ private BlobContainerClient GetContainer(string containerName)
         return container;
     }
 
-    public string UploadJson(object json, string? name = default)
+    public string UploadJson(object json, string? name = default, bool overwrite = false)
```

Comment:
could you also set content type in the blob metadata (same with upload bytes).

Reply:
Will do, but we don't know the content type with upload bytes.

Reply:
bytes should be octet stream by default. If the user wants something else, they should pass BinaryData with ContentType set